client/search-ui/src/results/sidebar/SearchFilterSection.module.scss

```
@@ -73,17 +73,11 @@
     }
 
     &__list-item-label {
-        :global(.core-workflow-improvements-enabled) & {
-            color: var(--body-color);
-        }
+        color: var(--body-color);
     }
 
     &__icon {
-        margin: 0 0.125rem;
-
-        :global(.core-workflow-improvements-enabled) & {
-            margin-left: 0;
-        }
+        margin: 0 0.125rem 0 0;
```

Would simplifying like this do the same?

```suggestion
        margin-right: 0.125rem;
```